k-distribution/tests/regression-new/markdownSelectors/test.md

```diff
@@ -24,6 +24,11 @@ module TEST
   rule <k> 1 => 2 </k>
 ```
 
+
+```{.k .discard .numberLines startFrom="0"}
+  rule <k> 2 => 3 </k>
+```
+
```

Comment:
This rule will not interact with the Makefile in the way I think you intend it to; the selector for `kompile` is:
```
k|(keep&!discard)
```
which means that because this rule has a `k` tag, it's selected and the `discard` tag doesn't need to be considered.

You can test this locally even if you can't build the Haskell backend by running `make -n` and running the `kompile` and `krun` commands it shows you.

Reply:
What do you think about changing the MD selector to `'(k&!discard)|(keep&!discard)'` ?
This still will execute the correct cells while preserving the idea of the test.